src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -182,13 +255,27 @@ function WorkspaceTaxesPage({policy, route}: WorkspaceTaxesPageProps) {
                         canSelectMultiple
                         sections={[{data: taxesList, indexOffset: 0, isDisabled: false}]}
                         onCheckboxPress={toggleTax}
-                        onSelectRow={() => {}}
+                        onSelectRow={(tax: ListItem) => tax.keyForList && Navigation.navigate(ROUTES.WORKSPACE_TAXES_EDIT.getRoute(policyID, tax.keyForList))}
```

Comment:
We should also deselect all taxes when navigating the user to the edit page, like we do for [categories](https://github.com/Expensify/App/blob/94a3111ea0369c1ff917e212f1b0d72cfa4f97fe/src/pages/workspace/categories/WorkspaceCategoriesPage.tsx#L141-L144)